packages/frontend-2/pages/dashboard.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="flex flex-col gap-y-12">
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">Quick start</h2>
+      <div class="flex flex-col md:grid md:grid-cols-3 gap-3 pt-5">
+        <QuickStartCard
+          v-for="(quickStartItem, index) in quickStartItems"
+          :key="`quick-start-${index}`"
```

Comment:
https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#af99b815b2a6430da927ca10a1eb9007

Adding a string suffix doesn't change the fact that the key is not a valid key because it doesn't uniquely identify the item. Each of those items has a unique title, why not use that?